### ISSUE TITLE
Lock screen: No way to get password prompt

### ISSUE THREAD

**Issue by spxak1** [pop-os/pop#1023]
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

**Issue/Bug Description:**
What happens is no combination of keys, and/or mouse clicks make the password prompt appear, as if the input devices are off. Then, as I keep pressing any key/mouse button in frenzy, it appears, enter the password and log in normally. The mouse pointer appears and moves normally. No icon is clickable while this happens (as if key presses and mouse clicks do not register). As soon as the password prompt appears, I can also click the icons at the top bar.

The problem is there when the screen dims too (no lock screen). You can see the mouse moving, but no clicks or key presses register to wake up the screen. 

**Steps to reproduce (if you know):**
Boot computer.
Log in normally.
Lock screen (either from menu or through timeout)
Mouse clicks and key presses don't bring up the password prompt, no icons are clickable.
Press keys at random (no key combination) or keep clicking until prompt comes up and icons become clickable again. 

**Expected behavior:**
From the lock screen: a single key press or mouse motion/click should bring the password prompt.
From dimmed screen: a single key press or mouse motion/click should get you back to desktop.

**Other Notes:**
Worked fine in 19.04
I have tested different keyb/mouse, same behaviour with USB,BT and laptops touchpad/keyb.

Also: lock screen icons at top right are default gnome, no Pop theme.

**Comment by leviport** [comment]
The issue is unclear. Could you please provide more details?

**Comment by spxak1** [comment]
Sorry, poor markup skills. Does it appear now?

**Comment by spxak1** [comment]
Solved!
The reason was that with 19.10 I had compiled and install lifprint (fingerprint sensor drivers) for my T460p, and as such my fingerprint sensor worked well.

I had installed libpam-fprintd and fprintd and in Settings I had the fingerprint sensor enabled.

With 20.04, sadly libfprint update to a new version that does not support my drivers (or any drivers for my device yet), so what was happening was that when the system entered the lock screen, it was waiting for fprintd to time out before it released the password prompt.

I removed fprintd and libpam-fprintd, and now the password prompt appears right away.

I figured it out from the fact that in Settings, when I opened Users, it was taking a long time (or would crash) to open that tab. A quick look at the journal revealed the timeout of fprintd.

Now all work great. 

Thanks for your time and effort, all the best.
